Compressed overview on resources (#1069)
inductiva/inductiva: docs/introduction/computational_resources_overview.md

# Overview

In this guide, we will explain some of the main features of the Inductiva API when 
it comes to making informed decisions about resource allocation. Here, you will 
learn the two options available for running your simulations; shared and dedicated
resources. Additionally, we will demonstrate the performance differences between 
the two resources using a SWASH simulation.

## Shared Resources

As a standard practice, when you submit simulation tasks through the Inductiva API, 
they enter a [task queue]() and are sent to a shared pool of workers on designated Virtual Machines (VMs)
serving multiple users from either the [Google Cloud Provider (GCP)](https://cloud.google.com/compute/docs/machine-resource) or Inductiva's own 

> computational platform (ICE). These VMs are specifically allocated to facilitate easy 

Reviewer: or, in a future version, Inductiva's...

Author: got it

Reviewer 2: I think this meant to be for the Inductiva's own computational platform (ICE) above.

API testing and to handle light tasks efficiently with a minimal setup, ideal for 
quick experimentation.

However, this shared resource pool has its limitations, including slower completion 
times for simulations due to its finite capacity and less powerful VMs. For running 
a larger volume of simulations, the Inductiva API provides you with the option to 

> set up your own [dedicated resources](). 

Reviewer 2: This may not need a link, since we talk about it right afterwards.

## Dedicated Resources

To ensure you can access the necessary computational power for your simulations 

> without the constraints of a shared environment, Inductiva provides the option 

Reviewer: 'resources' instead of 'environment' ? maybe..?

> to create dedicated pools of VM resources, termed [Machine Groups](), exclusively 

Reviewer 2: remove `termed [Machine Groups]()` since there are several options and mentioned them already below.

reserved for your use and not shared with others. 

> In this option, you can launch three types of computational resources for your 

Reviewer 2: ```suggestion
There are three types of dedicated computational resources you can launch:
```

## Comparison Example: SWASH Simulation

To illustrate the performance differences between running simulations on a shared 
pool of resources and on dedicated resources using the Inductiva API, we will run 

> the [SWASH simulation]() as an example.

Reviewer: where should this link to?

Author: either our own link to the swash in the simulator section, or the repo of the peeps behind swash? https://gitlab.tudelft.nl/citg/wavemodels/swash

Reviewer: I think it should be an internal link to our SWASH page, but it would be fine to add the link of the developers from our internal swash page.

Reviewer 2: I think it should point to the SWASH simulator, but just the SWASH term `[SWASH](../simulators/SWASH)`
